### DISCUSSION TITLE
*how can I install the security onion gui desktop to my VM?

### DISCUSSION THREAD

**Discussion by iqworks** [Security-Onion-Solutions/securityonion#10586] [Unsupported Versions]
I am using VMware Workstation. 
 Hi,  I am using VMware Workstation. 
  I am using SO version – 2.3.250-20230519.
  I would like to install the gui desktop, but I cannot find out how to install it. 

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/5569263/e3c2203d-59e8-4082-bde5-8d584f36a92c)

not sure if this is SO related, but, the cursor does not appear in the terminal window. Perhaps this is VMware's issue. I am just not sure.

I see the cursor in other VM’s I have created. Perhaps it could be this latest version of security onion?

Thanks for any help

**Comment by NetwarSystem** [comment]
Experienced Linux/VirtualBox user here. I am working with this ISO:

securityonion-2.3.250-20230519.iso

And on install if I pick the "other stuff" option, then choose analyst workstation, the install runs, but after reboot it comes up in text mode. If I log in it reports that startx is missing.

So this is my first visit, reviewing to see if this problem is already solved, and seems like maybe not ...

**Reply by dougburks** [reply]
@NetwarSystem I've replied to your separate discussion over at https://github.com/Security-Onion-Solutions/securityonion/discussions/10588.

**Comment by dougburks** [comment]
> I see the cursor in other VM’s I have created. Perhaps it could be this latest version of security onion?

@iqworks I don't believe this is an issue specific to Security Onion. Looking at your screenshot, it appears that you have successfully installed the Security Onion GUI desktop in your VMware VM and it looks like it has gone to the lock screen. 

Please try the following:
- click into the VM window
- press the Enter or Esc key

If that doesn't help, then try rebooting the VM.

**Reply by iqworks** [reply]
Sorry Doug, that screen shot is from a prevous version (2.3.190) of SO. The following is the screen shot of the new version 2.3.250-20230519.![so-current screen](https://github.com/Security-Onion-Solutions/securityonion/assets/5569263/6576a414-cc89-428e-a978-2647801027e4)
I would like to install the GUI on 2.3.250-20230519. 
  
Thanks as always

**Reply by iqworks** [reply]
found it!! sudo "so-analyst-install" gave me a GUI.
